hicommonwealth/commonwealth
# Added custom table sorting for stake transactions history

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6676

## Description of Changes
- Adds custom table sorting behaviour to stake transaction history tab

## "How We Fixed It"
N/A

## Test Plan
- Enable `FLAG_MY_COMMUNITY_STAKE_PAGE_ENABLED` in `.env`
- Visit `/myCommunityStake` -> "Transaction History" tab
- Verify you can filter the transaction table and each column shows correctly filtered values
- Verify you can sort multiple columns using `Shift + Enter/Click` on sort direction button

## Deployment Plan
N/A

## Other Considerations
N/A
